src/Text/Pandoc/Writers/Markdown.hs

```diff
@@ -408,6 +432,9 @@ blockToMarkdown' opts b@(RawBlock f str) = do
       | f `elem` ["markdown", "markdown_github", "markdown_phpextra",
                   "markdown_mmd", "markdown_strict"]
          -> return $ literal str <> literal "\n"
+    Markua -> do
+        report $ NoTranslation "raw content is writen out in original form and"
```

Comment:
I don't understand this...  NoTranslation has a specific use, to indicate that there is no translation for a given term.
I'm not sure what you're trying to warn about here, but another LogMessage constructor should be used.

Reply:
We used NoTranslation, because it seemed to be the most fitting, since markua doesn't support raw element. It is now changed to the "renderEmpty" you suggested, which is more fitting.